### PR TITLE
Separate confidence distributions histograms

### PR DESCRIPTION
**Pull request by jmrf**
**Proposed changes**:
- Change `evaluate.py` script to allow for separate histogram bars plotting wrong and correct intent predictions confidence distributions.

Changes related to issue #1354

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [x] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
Thanks a lot for submitting this PR! Before I review it, I actually think we should just always plot with two colours, and get rid of  the cmdline arg. Could you update the PR accordingly?

**Comment by jmrf** [comment]
Sure, no problem!

**Comment by akelad** [comment]
Thanks! We'll give it a review soon :)

File: rasa_nlu/evaluate.py [hunk 1]
```diff
@@ -122,18 +122,23 @@ def plot_confusion_matrix(cm, classes,
...
 
 
-def plot_histogram(hist_data,
-                   out=None):  # pragma: no cover
-    """Plot a histogram of the confidence distribution of the predictions.
-
+def plot_histogram(hist_data, out=None):    # pragma: no cover
+    """Plot a histogram of the confidence distribution of the predictions in
+    two columns.
+    Green for the confidences of hits, Red for the confidences of misses.
```
Comment (tmbo) [inline rasa_nlu/evaluate.py]: Please use two colors that can be read by people with deuteranopia.

i'd suggest
- positive: #009292 (wine-ish)
- negative: #920000 (blueish)

Comment (jmrf) [reply rasa_nlu/evaluate.py]: Very good point!
Ofc, the change is in its way

**Pull request merged**